app/api/business/case_study_business.py

```diff
@@ -0,0 +1,9 @@
+from app.api.services import (
+    case_study_service
+)
+
+
+def get_approved_case_studies(supplier_code, domain_id):
+    case_studies = case_study_service.get_approved_case_studies_by_supplier_code(supplier_code, domain_id)
+    case_studies = case_studies[0]
```

Comment:
This looks a bit odd when multiple case studies could be returned by the service.  An alternative might be to return the case studies as a list with each dictionary containing all the data relevant to the case study/evidence. Then you wouldn't need to refer to `case_studies[0]` before returning it.
```python
case_studies = [{'id': 123, 'data': {...}}, {'id': 456, 'data': {...}}]
```